docs/docs/about/comparison-powershell.md

```diff
@@ -305,6 +307,7 @@ Enable-PnPCommSite|PnP.PowerShell|[spo site commsite enable](../cmd/spo/site/sit
 Enable-PnPFeature|PnP.PowerShell|[spo feature enable](../cmd/spo/feature/feature-enable.md)
 Enable-PnPFlow|PnP.PowerShell|[spo flow enable](../cmd/flow/flow-enable.md)
 Enable-PnPPageScheduling|PnP.PowerShell|
+Enable-PnPPowerShellTelemetry|PnP.PowerShell|[cli config set](../telemetry/#re-enable-telemetry)
```

Comment:
According to the docs, the command is `cli config reset`

Reply:
yep